crates/hdi/README.md

```diff
@@ -9,92 +9,7 @@
-
-**The coordination zomes comprise the application's controller layer** — the code that actually
-writes and retrieves data, handles countersigning sessions and sends and receives messages
-between peers or between a cell and its UI. In other words, all the zome functions, `init`
-functions, remote signal receivers, and scheduler callbacks will all live in coordinator zomes.
-
-Advantages of this approach are:
-* The DNA hash is constant as long as the integrity zomes remain the same. The peer network of
-a DNA is tied to its hash. Changes to the DNA hash result in a new peer network. Changes to the
-domain logic enclosed in coordinator zomes, however, do not affect the DNA hash. Hence the DNAs
-and therefore hApps can be modified without creating a new peer network on every
-deployment.
-* Integrity zomes can be shared among DNAs. Any coordinator zome can import an integrity
-zome's data types and implement functions for data manipulation. This composability of
-integrity and coordinator zomes allows for a multitude of permutations with shared integrity
-zomes, i. e. a shared data model.
-
-## Data validation
-
-The second fundamental part of integrity zomes is data validation. For every [operation](holochain_integrity_types::Op)
-that is produced by an [action](holochain_integrity_types::Action), a
-validation rule can be specified. Both data types and data values can be
-validated.
-
-All of these validation rules are declared in the `validate` callback. It
-is executed for a new action by each validation authority.
-
-There's a helper type called [`OpType`](holochain_integrity_types::OpType) available for easy
-access to all link and entry variants when validating an operation. In many cases, this type can
-be easier to work with than the bare [`Op`](holochain_integrity_types::Op), which contains the
-same information as `OpType`, but the former has a flatter data structure, whereas the latter has
-a deeply nested structure.
-
-```rust
-match op.to_type()? {
-    OpType::StoreEntry(OpEntry::CreateEntry { app_entry, .. }) => match app_entry {
-        EntryTypes::A(_) => Ok(ValidateCallbackResult::Valid),
-        EntryTypes::B(_) => Ok(ValidateCallbackResult::Invalid(
-            "No Bs allowed in this app".to_string(),
-        )),
-    },
-    OpType::RegisterCreateLink {
-        base_address: _,
-        target_address: _,
-        tag: _,
-        link_type,
-    } => match link_type {
-        LinkTypes::A => Ok(ValidateCallbackResult::Valid),
-        LinkTypes::B => Ok(ValidateCallbackResult::Invalid(
-            "No Bs allowed in this app".to_string(),
-        )),
-    },
-    _ => Ok(ValidateCallbackResult::Valid),
-};
-```
-See an example of the `validate` callback in an integrity zome in the WASM workspace:
-<https://github.com/holochain/holochain/blob/develop/crates/test_utils/wasm/wasm_workspace/validate/src/integrity.rs>.
-Many more validation examples can be browsed in that very workspace.
+<!-- cargo-rdme -->
```

Comment:
It doesn't appear to be working.

Reply:
It's working, just that the files are not yet generated. You can run the script locally to test it.
I've added the generated files anyway.